src/actions/queries.js

```diff
@@ -48,11 +48,14 @@ export const getUserFromId = (id) => axios.get(`/api/users/${id}`);
 
 export const deleteUser = (id, user) => axios.delete(`/api/users/${id}`, user);
 
-export const getWaivers = () => axios.get("/api/waivers?adult=true&minor=true");
+//WAIVERS
+export const getWaivers = (type, organizationId) =>
+  axios.get(`/api/waivers&type=${type}&organizationId=${organizationId}`);
 
-export const deleteWaiver = (id) => axios.delete(`/api/waivers/${id}`);
+export const deleteWaiver = (type, organizationId) =>
+  axios.delete(`/api/waivers&type=${type}&organizationId=${organizationId}`);
```

Comment:
careful with how these line up with the server. You need to include the id in the URL like `waivers/${id}` for it to route to the `api/waivers/[id].js` file. For you, I think it can just go to the index.js file (so keep the queries.js as you changed id), but that means you need to move your handler code from `[id].js` to `index.js`

Reply:
Why is there a deleteWaiver query? Shouldn't it just be the getWaivers and uploadWaiver, where we could pass an empty string to remove the text (also please change the second one to updateWaiver)